main.go

```diff
@@ -359,6 +361,27 @@ func (s *admissionWebhookServer) addResources(c *corev1.Container, r map[string]
 	}
 }
 
+func (s *admissionWebhookServer) addNscLimits(c *corev1.Container) {
+	s.createResourcesLimits(c, s.config.NscLimitsCPU, s.config.NscRequestsCPU, s.config.NscLimitsMemory, s.config.NscRequestsMemory)
+}
+
+func (s *admissionWebhookServer) addNscInitLimits(c *corev1.Container) {
+	s.createResourcesLimits(c, s.config.NscInitLimitsCPU, s.config.NscInitRequestsCPU, s.config.NscInitLimitsMemory, s.config.NscInitRequestsMemory)
+}
+
+func (s *admissionWebhookServer) createResourcesLimits(c *corev1.Container, limitsCPU, requestsCPU, limitsMemory, requestsMemory string) {
+	c.Resources = corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{
+			"cpu":    resource.MustParse(limitsCPU),
```

Comment:
```suggestion
			"cpu":    resource.MustParse(s.config.LimitsCPU),
```

Reply:
What is the idea behind this change?

Reply:
As discussed personally, removed separate envs for the init container